src/DataStreams/CsvStream.php

```diff
@@ -196,7 +196,6 @@ private function readCsvLine($trimValues = false)
                 switch ($byte) {
                     case $this->enclosure:
                         $inEnclosure = !$inEnclosure;
-                        continue;
```

Comment:
For prosperity, this continue is unneeded as the break on the next line is the last line executed in the loop anyway.